charts/scalardb-cluster/README.md

```diff
@@ -25,6 +25,12 @@ Current chart version is `2.0.0-SNAPSHOT`
 | fullnameOverride | string | `""` | String to fully override scalardb-cluster.fullname template |
 | nameOverride | string | `""` | String to partially override scalardb-cluster.fullname template (will maintain the release name) |
 | scalardbCluster.affinity | object | `{}` | The affinity/anti-affinity feature, greatly expands the types of constraints you can express. |
+| scalardbCluster.encryption.enabled | bool | `false` | Enable encryption feature. You must enable encryption when you use the encryption feature of ScalarDB Cluster. |
+| scalardbCluster.encryption.type | string | `""` | Type of encryption. You must set the same value as "scalar.db.cluster.encryption.type" of ScalarDB Cluster. |
+| scalardbCluster.encryption.vault | object | `{"tls":{"caRootCertSecret":"","enabled":false}}` | Vault specific configurations. |
+| scalardbCluster.encryption.vault.tls | object | `{"caRootCertSecret":"","enabled":false}` | TLS configurations to access from ScalarDB Cluster to Vault by using TLS. |
```

Comment:
It's more like a question to @brfrn169.
I don't see TLS-related configurations in the current doc, but are they going to be added?
https://github.com/scalar-labs/docs-internal-scalardb/pull/452

Reply:
Ah, I see. I overlooked it.
I will create a PR to add those TLS configurations!

Reply:
I created the PRs!
- https://github.com/scalar-labs/docs-internal-scalardb/pull/488
- https://github.com/scalar-labs/docs-internal-scalardb/pull/489

Reply:
@kota2and3kan Thanks!